flocker/volume/functional/test_filesystems_zfs.py

```diff
@@ -527,10 +527,9 @@ def test_maximum_size_enforced(self):
         """
         pool = build_pool(self)
         service = service_for_pool(self, pool)
-        # There is a lower-bound on the value of refquota in ZFS.  It seems to
-        # be 64MB (but perhaps this isn't universal).
+        maximum_size = 40 * 1024 * 1024
```

Comment:
Use the `bitmath` library to write this as `40 * MiB`. It's a minor change, but it makes things more obvious - easier to see if you meant GiB, for example.